cpp/include/cudf/dictionary/dictionary_factories.hpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/column/column.hpp>
+#include <cudf/column/column_view.hpp>
+
+
+namespace cudf
+{
+
+/**
+ * @brief Construct a dictionary column by copying the provided `keys`
+ * and `indices`.
+ *
+ * It is expected that `keys_column.has_nulls() == false`.
+ * It is assumed the elements in `keys_column` are unique and
+ * are in a strict, total order. Meaning, `keys_column[i]` is _ordered before
```

Comment:
errant underscore? It's in all of your comments that have this phrase.

```suggestion
 * are in a strict, total order. Meaning, `keys_column[i]` is ordered before
```

Reply:
Found and fixed all 3 occurrences.